tests/e2e/app/authn-central-login/autoscript.js

```diff
@@ -18,15 +18,24 @@
   const url = new URL(window.location.href);
   const amUrl = url.searchParams.get('amUrl') || 'https://auth.example.com:9443/am';
   const code = url.searchParams.get('code') || '';
-  const clientId = url.searchParams.get('clientId') || 'WebOAuthClient';
+  const clientId = url.searchParams.get('clientId') || 'CentralLoginOAuthClient';
```

Comment:
This fixes the e2e failures as I can now isolate this test from the other OAuth tests.

Reply:
👍 great!